independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/app/CuratedApplication.java

```diff
@@ -208,28 +208,20 @@ public synchronized QuarkusClassLoader getBaseRuntimeClassLoader() {
                 //in test mode we have everything in the base class loader
                 //there is no need to restart so there is no need for an additional CL
 
-                for (AdditionalDependency i : quarkusBootstrap.getAdditionalApplicationArchives()) {
-                    //src/test is the highest priority
-                    if (i.isTestClassRoot()) {
```

Comment:
So, the test paths are now expected to be among the app root paths, correct?

Reply:
Yes, there is no special case handling for them any more, they are just installed first so their application.properties takes precidence.